### ISSUE TITLE
a11y:  Example table: Hidden scrollable content cannot be scrolled using the keyboard

### ISSUE THREAD

**Issue by cjfreedman** [swagger-api/swagger-ui#9582]
### Q&A (please complete the following information)
 - OS: [e.g. macOS]  Windows 10
 - Browser: [e.g. chrome, safari]  Chrome
 - Version: [e.g. 22]  121
 - Method of installation: [e.g. npm, dist assets]  npm
 - Swagger-UI version: [e.g. 3.10.0]   Swagger UI React 5.11.3
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0] OpenAPI 3.0

### Content & configuration
Basic/default configuration

### Describe the bug you're encountering
We are using a tool Sort Site to evaluate our site for accessibility issues.  it is flagging the Request body Example table: "Hidden scrollable content cannot be scrolled using the keyboard.  Hidden scrollable content must either contain a focusable element like a link or have "tabindex=0" set.  This allows keyboard users to tab to the hidden content and scroll using the arrow keys. "  Violates wCAg 2.0 A 2.1.1 https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html
This is happening on element "pre" with class "body-param__example microlight"

### To reproduce...
Click to expand a method pane like a POST method, and see the example tab.  The reported error is occurring there.

### Expected behavior
No accessibility issue

### Screenshots
Here's the area on the screen
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/bbc616d1-48e4-41ef-8baf-7d60542c62f8)

Here's the Sort Site error:
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/c19cd2f3-5129-4f8f-9aa3-0fcf6fe8d253)
